### ISSUE TITLE
Is this repo actually used for PRs?

### ISSUE THREAD

**Issue by jonathanunderwood** [openwrt/firewall4#39]
There are currently 9 PRs open, with the eearliest being > 9 months old, and seemingly no response from @jow-. This makes me wonder if this is actually a viable route to contribute to fw4? @jow- (or anyone else with commit access) any chance you could provide some clarification on your preferred mechanism for contributing changes to fw4?

**Comment by jow-** [comment]
Yes it is. Unfortunately @brada4's PR descriptions are often too terse to understand, some open PRs are wip and/or obsoleted by later PRs and/or changed from time to time.

Du to that, merging those has to wait until I have more spare time at my hand to actually review and test them.

**Comment by jonathanunderwood** [comment]
Thanks @jow- - fully understand the time-starved aspect, but it's reassuring that the PRs are in the right venue!
